### PR TITLE
M sc1 59 19

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: mskokova/05_HTML/task_01.html [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+					</FIELDSET>
+				</FORM>
+			</td>
+			<td>
+				<FORM method="post" action="http://sc.it-academy.by/fe/form.php">
+					<FIELDSET>	
+						<LEGEND>Строка поиска</LEGEND>
+						<INPUT TYPE="text"  NAME="text" placeholder="Только русские буквы"lang="ru"><br>
+						<INPUT TYPE="text" ID="index" NAME="index"
+						pattern="\+375-(29|44|25)-[0-9]{7}"placeholder="+375-29-1234567">
```
Comment (pluhin) [inline mskokova/05_HTML/task_01.html]: Wrong pattern

File: mskokova/05_HTML/task_01.html [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+			</td></tr>
+			<tr><td><STRONG>Загрузка файла на сервер:</STRONG></td>
+			<td><STRONG>Валидация данных HTML5:</STRONG></td></tr>
+			<tr><td>
+				<FORM method="post"enctype="multipart/form-data"action="http://sc.it-academy.by/fe/form.php">
+					<FIELDSET>
+						<LEGEND>Upload</LEGEND>
+						Файл:<br>
+						<input type="file" name="file" dir="ltr"><br>
+						Срок хранения (года):<br>
```
Comment (pluhin) [inline mskokova/05_HTML/task_01.html]: Where is label?

File: mskokova/05_HTML/task_01.html [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+					</FIELDSET>
+				</FORM>
+			</td></tr>
+			<tr><td><STRONG>Загрузка файла на сервер:</STRONG></td>
+			<td><STRONG>Валидация данных HTML5:</STRONG></td></tr>
+			<tr><td>
+				<FORM method="post"enctype="multipart/form-data"action="http://sc.it-academy.by/fe/form.php">
+					<FIELDSET>
+						<LEGEND>Upload</LEGEND>
+						Файл:<br>
```
Comment (pluhin) [inline mskokova/05_HTML/task_01.html]: Where is label?

File: mskokova/04_HTML/task_4_3.html [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
...ajpYcL1CwOJkO/YAQ5KgborsbhWxcF+C7EwB6MEhBDoGkFLmGN6vgxdo6h7Ck0WhV3XsJJxuzdk/PbxkegjNjQ2xQF11goFD787ECBRC6i5ChslD/t7A+iMGdDY8xgTQmAaFJGAhrm0A6mAqiuhszoWslIvV4Ur6+s/a9KjLUpXA5FQmEZ8Hoo2hcYI+l/xoavNAKMvv0hKCDqadQwmAvjh9wKm0mEUHT9sWJTIXLLgdcUpMVQ7CInHggROXaLsCHS3Gdi72/+v0Bdld6uBfYkmOJ4PT20D0AJQRGunnMcpFPygPODnLvK2RGtER7LLv838nYRRguF+DaMHnrtJMd2kIEFtS6lY5cgUFNSKh3KtiKDJcGS/BZ/eOCBvS+RtwFMUnU2AjwPf/AQ8yhJkCgAhAFSDpSCophRx4LmVjG3CFj48ygM3l4FdEYqRJODTG8rfTVF8m2qMv/4ESHYCd1aAP5a3bttAUlGrUng1zRPumsZIulCRPYKYz8y0XGBbxqudXaIxgAIoOzUoiCyR9hoVBSsLkNuytik5U9D/bpyp7bR/cblPE/BzAZ8moGsCqr4hOaoPonJ5lTZKn3tFOPkCFzm0+G20+m1EjQoqNYmSI5G3JXy0ip5ICT2REoTwUHIkIkYV7VZDn4scZCVna17mml8+2WQA8GZv8XHOeL2HwBuKt2gIGR4oXMyla5h66ODHO2X8uV6DxV1YuoulTA3TC1UU7RpM5kEjLtKrG6jauasD9Ptx13WzDABmZ2dV28FTi/W6elVI0dXZzIipE0gpIaQEpwoGVzC0Rg88nydwMbdSlOvr+akWMXMpytbuxePx0rPqdno0vrFQbJ4qOyxZrHjdQT9HU1CDzil0jbzUOAPWCy7uLhawmilOtXrTFzowOTk/P78+Pj4unoFTqZQaSdQ2C0byVtGGP5Ovd5dtofu4RwKmBo1SUAIoJbCRszGzWJGz6VI5lytebREzlzowOWlZVvby5cv/fE8NOXfuHE2n04F79MSROgseJ5IlodH2oMlMACjboiqVzHJUH2he5lqSXf+VEJK5f/9+/sUfZKe8JWfOnNGXnF2RgtcVb+Q+CSoISuHViLTXonJ51S+fbEopS729vc758+fli4C/AAAdTsx0wjCZAAAAAElFTkSuQmCC" alt="домой" title="домой"></a></center>
```
Comment (pluhin) [inline mskokova/04_HTML/task_4_3.html]: Do not use tag center, it is veeeeery old,

**Review by pluhin — CHANGES_REQUESTED** [review]

File: mskokova/05_HTML/task_01.html (hunk 1 above)
Comment (ghost) [reply mskokova/05_HTML/task_01.html]: Thanks. Do you mean should be without "-"? Like +375(29|44|25)[0-9]{7}? 
Cause I misunderstand the task.

File: mskokova/05_HTML/task_01.html (hunk 3 above)
Comment (ghost) [reply mskokova/05_HTML/task_01.html]: Oh, that label!

**Comment by pluhin** [comment]
You have conflicts, could you please solve them

**Pull request merged**